tests/dummy/app/templates/components/rows/selectable-table.hbs

```diff
@@ -1,40 +1,62 @@
 {{! BEGIN-SNIPPET selectable-table }}
-<div class="table-actions">
+<div class='table-actions'>
   {{#if this.hasSelection}}
     <div
-      class="table-action fa fa-check-square"
```

Comment:
Same with all the `fa` instances in this file. They should still work and we shouldn't change them.